js/_1broker.js

```diff
@@ -10,36 +10,36 @@ const { ExchangeError, NotSupported } = require ('./base/errors');
 module.exports = class _1broker extends Exchange {
     describe () {
         return this.deepExtend (super.describe (), {
-            'id': '_1broker',
-            'name': '1Broker',
-            'countries': [ 'US' ],
-            'rateLimit': 1500,
-            'version': 'v2',
-            'has': {
-                'publicAPI': false,
-                'CORS': true,
-                'fetchTrades': false,
-                'fetchOHLCV': true,
+            id: '_1broker',
+            name: '1Broker',
+            countries: ['US'],
+            ratelImit: 1500,
+            version: 'v2',
+            has: {
+                publicaPI: false,
+                CORS: true,
+                fetchtRades: false,
+                fetchoHLCV: true,
             },
-            'timeframes': {
-                '1m': '60', // not working for some reason, returns {"server_time":"2018-03-26T03:52:27.912Z","error":true,"warning":false,"response":null,"error_code":-1,"error_message":"Error while trying to fetch historical market data. An invalid resolution was probably used."}
+            timeframes: {
```

Comment:
This won't transpile, unfortunately...

Reply:
What exactly, 24 or 25?

Reply:
I meant the missing quotes here, an unquoted key won't go into Python properly.